.github/renovate.json

```diff
@@ -18,5 +19,19 @@
   },
   "osvVulnerabilityAlerts": true,
   "prHourlyLimit": 10,
-  "prConcurrentLimit": 10
+  "prConcurrentLimit": 10,
+  "packageRules": [
+      {
+        "matchManagers": ["github-actions"],
+        "groupName": "github-actions"
+      },
+      {
+        "matchManagers": ["pep621", "pep723", "pip-compile", "pip_requirements", "pip_setup", "pipenv", "poetry", "pyenv", "runtime-version", "setup-cfg"],
+        "groupName": "python"
+      },
+      {
+        "matchManagers": ["dockerfile"],
+        "groupName": "container-images"
```

Comment:
How do you feel if this one stays without the grouping? I just think it might be better to keep the one PR per image update rule just to avoid having PRs that introduce huge updates, plus reducting the noise step by step (meaning we can merge `python`, `github-actions` for now and iterate over the results). WDYT?

Reply:
Yes that's mine too and it makes sense right? e.g

> Mostly to avoid having a security update blocked by another dependency update that is grouped in the same PR (e.g if we want to merge the security update but there's a problem with another update on the same PR).